.gitignore

```diff
@@ -5,3 +5,5 @@ __pycache__/
 *.py[cod]
 *$py.class
 *.db
+
+.vscode/
```

Comment:
Project-specific files to be ignored belong in the project-specific `.gitignore`. This is not, and should be placed in your [global `.gitignore`](https://stackoverflow.com/q/7335420/4759433) instead (so please revert this change).